e2e/tests/node_not_ready/node_disconnection.robot

```diff
@@ -0,0 +1,143 @@
+*** Settings ***
+Documentation     https://longhorn.github.io/longhorn-tests/manual/pre-release/node-not-ready/node-disconnection/node-disconnection/
+...
+...               Keywords are imported from the resource file
+Resource          ../../keywords/node.resource
+Resource          ../../keywords/common.resource
+Suite Setup    set_test_suite_environment
+Test setup    set_test_environment    ${TEST NAME}
+Test Teardown    cleanup_resources
+
+*** Variable ***
+${volume_size_gb}=    1
+${retry_time_out}=    600
+${retry_interval}=    5
+
+*** Test Cases ***
+1-RWO 2 replica abnormal no attach no replica node data writing
```

Comment:
ping @weizhe0422

Reply:
> Do we need to prefix with `1-`, `2-`?
> 
> The numbering means it will be hard to add new cases between existing ones without adjusting the numbers of subsequent cases.

ping @weizhe0422

Reply:
I agree with you that the number prefix is ​​not necessary. Therefor, I've update the test case name, and it will be like `Node with replica and 3 replica volume attached` or `Node with replica but non-2 replica volume attached timeout and the volume attach on replica node`. 

In addition, thanks to @khushboo-rancher that she will help to give the proper name for each case.